src/dispatchers/mouseDispatcher.ts

```diff
@@ -2,110 +2,119 @@
 
 module Plottable {
 export module Dispatcher {
-  export class Mouse extends Dispatcher.AbstractDispatcher {
-    private _mouseover: (location: Point) => any;
-    private _mousemove: (location: Point) => any;
-    private _mouseout: (location: Point) => any;
+  export class Mouse {
+    private static dispatcherKey = "__Plottable_Dispatcher_Mouse";
```

Comment:
Understood.

Still please change the var name to be in all caps with underscores